### PR TITLE
Add section for detailing significant changes to the convention

### PR DESCRIPTION
**Pull request by gavinmacaulay**
I have added a section near the end of the document where we can detail and explain the changes that have been made to the document. I suggest we aim to update this as necessary whenever we commit changes to convention. 

@cyrilleponcelet could you edit that section to list the main changes that IFREMER have worked on? Thanks.

### CODE REVIEW

File: docs/crr341.adoc [hunk 1]
```diff
@@ -918,6 +918,16 @@ where latexmath:[\eta_\theta] is the angle sensitivity for the minor axis and la
 
 == Revision history
 
+=== Significant changes from version 1 to version 2
+
+Version 2 of this document makes some significant changes to the convention presented in version 1. These are summarised here:
+
+Beam pointing:: The method used to specify the beam pointing direction has changed from angles to vectors. A number of coordinate systems and translations and rotations have also been added to enable the flexible and complete definition of beam pointing directions.
+
+Platform attributes:: Some platform attributes (e.g., geographic position) have moved to different groups.
```
Comment (cyrilleponcelet) [inline docs/crr341.adoc:927]: Hi gavin, by platform attributes do you mean platform variables ? I do not clearly understand what this refers to. Is is referring to the fact that we added geographic position interpolated at time of ping in Sonar subgroups ?

Comment (gavinmacaulay) [reply docs/crr341.adoc:927]: yes, that's it - the interpolated positions at ping time (I couldn't quite remember what it was :))

Comment (gavinmacaulay) [reply docs/crr341.adoc]: @cyrilleponcelet - I have updated the PR - do my additions make sense?

**Comment by cyrilleponcelet** [comment]
@gavinmacaulay - last commit seems good to me and quite clear -

**Pull request merged**